splink/internals/linker_components/inference.py

```diff
@@ -124,6 +126,8 @@ def predict(
         threshold_match_probability: float = None,
         threshold_match_weight: float = None,
         materialise_after_computing_term_frequencies: bool = True,
+        num_chunks_left: int | None = None,
+        num_chunks_right: int | None = None,
```

Comment:
Maybe something for a separate PR, but I wonder if this left/right API makes the most sense? I feel like perhaps the concept of chunking should be tied to the individual datasets.

Take `dedupe_only`. Then probably you want to pass just one of these, and perhaps a bit confusing to have both options. And maybe even more confusing for multi-link case - would have to think for a moment what I would expect to happen when linking three sets what it would mean to chunk left/right by specified sizes. It seems more natural there to chunk per-dataset - say I have tables of sizes (100, 1M, 5M), I feel like I would prefer an option like (1, 5, 10) or something.